[2021-02-19 14:37:56,132] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: titanic_pivot.download_titanic_dataset 2021-02-19T11:34:10.907911+00:00 [queued]>
[2021-02-19 14:37:56,153] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: titanic_pivot.download_titanic_dataset 2021-02-19T11:34:10.907911+00:00 [queued]>
[2021-02-19 14:37:56,153] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-02-19 14:37:56,153] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-02-19 14:37:56,153] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-02-19 14:37:56,160] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): download_titanic_dataset> on 2021-02-19T11:34:10.907911+00:00
[2021-02-19 14:37:56,162] {standard_task_runner.py:52} INFO - Started process 57560 to run task
[2021-02-19 14:37:56,165] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'titanic_pivot', 'download_titanic_dataset', '2021-02-19T11:34:10.907911+00:00', '--job-id', '3', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/titanicTest.py', '--cfg-path', '/tmp/tmpmrx6_ek0', '--error-file', '/tmp/tmpl6vdzs3o']
[2021-02-19 14:37:56,165] {standard_task_runner.py:77} INFO - Job 3: Subtask download_titanic_dataset
[2021-02-19 14:37:56,201] {logging_mixin.py:104} INFO - Running <TaskInstance: titanic_pivot.download_titanic_dataset 2021-02-19T11:34:10.907911+00:00 [running]> on host oleg-desktop
[2021-02-19 14:37:56,286] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=titanic_pivot
AIRFLOW_CTX_TASK_ID=download_titanic_dataset
AIRFLOW_CTX_EXECUTION_DATE=2021-02-19T11:34:10.907911+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-02-19T11:34:10.907911+00:00
[2021-02-19 14:37:57,868] {python.py:118} INFO - Done. Returned value was: None
[2021-02-19 14:37:57,886] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=titanic_pivot, task_id=download_titanic_dataset, execution_date=20210219T113410, start_date=20210219T113756, end_date=20210219T113757
[2021-02-19 14:37:57,918] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-02-19 14:37:57,938] {local_task_job.py:146} INFO - Task exited with return code 0
